simpledotorg/simple-server
# Dashboard recent updates oct2022

**Story card:** ---

## Because

October updates

## This addresses

--

## Test instructions

Check the update text on the [home page of the dashboard](http://localhost:3000/my_facilities)
